README.md

```diff
@@ -25,5 +25,14 @@ To see a list of build options, run `gradle tasks` (or `gradlew tasks`). The `bu
 To build the SpotBugs plugin for Eclipse, you'll need to create the file `eclipsePlugin/local.properties`, containing a property `eclipseRoot.dir` that points to an Eclipse installation's root directory (see `.travis.yml` for an example), then run the build.
 To prepare Eclipse environment only, run `./gradlew eclipse`. See also [detailed steps](https://github.com/spotbugs/spotbugs/blob/master/eclipsePlugin/doc/building_spotbugs_plugin.txt).
 
+# Using SpotBugs
+
+SpotBugs can be used standalone and through several integrations, including:
+
+* [Ant](http://spotbugs.readthedocs.io/en/latest/ant.html)
+* [Maven](https://github.com/hazendaz/findbugs-maven-plugin/tree/spotbugs)
```

Comment:
This is the old page, maven plugin moved to spotbugs : https://github.com/spotbugs/spotbugs-maven-plugin

May be the link to the manual would be better: http://spotbugs.readthedocs.io/en/latest/maven.html